sdk/identity/identity-broker/samples-dev/index.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * @summary Demonstrates usage of @azure/identity-broker and @azure/identity
+ * packages for WAM MSA support.
+ */
+
+import {
+  InteractiveBrowserCredential,
+  InteractiveBrowserCredentialNodeOptions,
+  useIdentityPlugin,
+} from "@azure/identity";
+import { createNativeBrokerPlugin } from "@azure/identity-broker";
+import dotenv from "dotenv";
+import { BrowserWindow } from "electron";
+const win = new BrowserWindow();
+
+// case 1:
+//   - broker not loaded - when the options are set and the plugin not loaded
```

Comment:
Remove these comments, although you need to test for them.

Reply:
Tested the 1st scenario.
Can you test the javascript sample on Mac OS? @mpodwysocki

Reply:
Testing done.